src/components/core/geoChart/tests/geoChartColor.spec.ts

```diff
@@ -162,11 +162,11 @@ describe("getPushpinColors", () => {
         });
         const colorStrategy: IColorStrategy = createColorStrategy(executionResponse, geoConfig, segmentItems);
         const expectedColors: IPushpinColor[] = [
-            { background: "rgb(215,242,250)", border: "rgb(20,178,226)" },
-            { background: "rgb(212,244,236)", border: "rgb(0,193,141)" },
-            { background: "rgb(237,136,129)", border: "rgb(229,77,66)" },
-            { background: "rgb(241,134,0)", border: "rgb(241,134,0)" },
-            { background: "rgb(171,85,163)", border: "rgb(171,85,163)" },
+            { background: "rgba(215,242,250,0.7)", border: "rgba(20,178,226,0.7)" },
```

Comment:
please double check the color of `border`.
We don't need transparency for it :)

Reply:
As we also update the color palette and border take the darkest one in it, so border gets affected as well.